### DISCUSSION TITLE
Using Standalone .NET Aspire dashboard and messages aren't showing formatted message

### DISCUSSION THREAD

**Discussion by antmeehan** [microsoft/aspire#5131] [Q&A] [answered]
I've just spun up the [Standalone .NET Aspire dashboard](https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/dashboard/standalone?tabs=bash) (which is great!) but the messages in the structured logs aren't showing the formatted message:

![image](https://github.com/user-attachments/assets/324469ad-a971-41e9-9f70-81449d292c21)

For example, instead of `Latest local timestamp for {0} is '{1:O}'` I would expect the parameters to be included (ie `Latest local timestamp for SiteServiceConfiguration is '24/04/2024 11:01:06 AM +08:00'`

Any ideas?

**Comment by elfico** [comment]
For some reason, I couldn't get the logs to show. Traces and Metrics are showing though.
My logging set up
```
 logging.AddOpenTelemetry(opt =>
 {
     opt.SetResourceBuilder(ResourceBuilder.CreateDefault()
         .AddService("Flight Service"));
 });
 //logging.ClearProviders();
 logging.SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Information);
```

**Comment by jdaigle** [comment, marked as answer]
I ran into the same problem and eventually found an option, `IncludeFormattedMessage`, that can be set to true. For example:

```
logging.AddOpenTelemetry(opt =>
{
    opt.IncludeFormattedMessage = true;
});
```

**Reply by tibitoth** [reply]
Interesting, how can I include both formatted message and message template?
